+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Sat Dec  3 12:53:14 2022 - INFO  - Start to run test.
+ cat
+ openssl genrsa -out rsaprivatekey.pem -passout pass:trousers -des3 2048
oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh: line 28: openssl: command not found
+ CHECK_RESULT 127
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 29'
+ message='oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 29'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 29'
Sat Dec  3 12:53:15 2022 - ERROR - oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 29
+ return 0
+ grep 'BEGIN RSA PRIVATE KEY' rsaprivatekey.pem
grep: rsaprivatekey.pem: No such file or directory
+ CHECK_RESULT 2
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 31'
+ message='oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 31'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 31'
Sat Dec  3 12:53:16 2022 - ERROR - oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 31
+ return 0
+ openssl rsa -in rsaprivatekey.pem -passin pass:trousers -pubout -out rsapublickey.pem
oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh: line 32: openssl: command not found
+ CHECK_RESULT 127
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 33'
+ message='oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 33'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 33'
Sat Dec  3 12:53:17 2022 - ERROR - oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 33
+ return 0
+ grep 'BEGIN PUBLIC KEY' rsapublickey.pem
grep: rsapublickey.pem: No such file or directory
+ CHECK_RESULT 2
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 35'
+ message='oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 35'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 35'
Sat Dec  3 12:53:18 2022 - ERROR - oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 35
+ return 0
+ openssl rsautl -encrypt -pubin -inkey rsapublickey.pem -in plain.txt -out cipher.txt
oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh: line 36: openssl: command not found
+ CHECK_RESULT 127
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 37'
+ message='oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 37'
Sat Dec  3 12:53:19 2022 - ERROR - oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 37
+ return 0
+ test -f cipher.txt
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 39'
+ message='oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 39'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 39'
Sat Dec  3 12:53:19 2022 - ERROR - oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 39
+ return 0
+ expect
spawn openssl rsautl -decrypt -inkey rsaprivatekey.pem -in cipher.txt -out plain_back.txt
couldn't execute "openssl": no such file or directory
    while executing
"spawn openssl rsautl -decrypt -inkey rsaprivatekey.pem -in cipher.txt -out plain_back.txt"
+ grep rsaprivatekey.pem testlog1
spawn openssl rsautl -decrypt -inkey rsaprivatekey.pem -in cipher.txt -out plain_back.txt
"spawn openssl rsautl -decrypt -inkey rsaprivatekey.pem -in cipher.txt -out plain_back.txt"
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ diff plain.txt plain_back.txt
diff: plain_back.txt: No such file or directory
+ CHECK_RESULT 2
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 51'
+ message='oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 51'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 51'
Sat Dec  3 12:53:21 2022 - ERROR - oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 51
+ return 0
+ expect
spawn openssl rsautl -sign -inkey rsaprivatekey.pem -in plain.txt -out signature.bin
couldn't execute "openssl": no such file or directory
    while executing
"spawn openssl rsautl -sign -inkey rsaprivatekey.pem -in plain.txt -out signature.bin"
+ grep rsaprivatekey.pem testlog2
spawn openssl rsautl -sign -inkey rsaprivatekey.pem -in plain.txt -out signature.bin
"spawn openssl rsautl -sign -inkey rsaprivatekey.pem -in plain.txt -out signature.bin"
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ test -f signature.bin
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 63'
+ message='oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 63'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 63'
Sat Dec  3 12:53:22 2022 - ERROR - oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 63
+ return 0
+ openssl rsautl -verify -pubin -inkey rsapublickey.pem -in signature.bin -out signature_verify.bin
oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh: line 64: openssl: command not found
+ CHECK_RESULT 127
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 65'
+ message='oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 65'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 65'
Sat Dec  3 12:53:23 2022 - ERROR - oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 65
+ return 0
+ test -f signature_verify.bin
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 67'
+ message='oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 67'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 67'
Sat Dec  3 12:53:23 2022 - ERROR - oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 67
+ return 0
+ diff plain.txt signature_verify.bin
diff: signature_verify.bin: No such file or directory
+ CHECK_RESULT 2
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 69'
+ message='oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 69'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 69'
Sat Dec  3 12:53:24 2022 - ERROR - oe_test_openssl_encrypt_decrypt_file_asymmetrically.sh line 69
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Sat Dec  3 12:53:25 2022 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 11
+ test 11 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Sat Dec  3 12:53:26 2022 - ERROR - failed to execute the case.
+ exit 11
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Sat Dec  3 12:53:27 2022 - INFO  - Start to restore the test environment.
++ ls
++ grep -v '\.sh\|common'
+ rm -f plain.txt testlog1 testlog2
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Sat Dec  3 12:53:27 2022 - INFO  - End to restore the test environment.
